roles/k3s/pull-config/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Copy kube config for adjustments
+  copy:
```

Comment:
I think we're missing the file permissions here

Reply:
Went ahead and set the mode on the copy. When k3s is installed it sets the ~/.kube/config to 644. The file permissions seem to stick on the copy. I am not sure if it's how I have configured my cloud-init Templates. Great for fault tolerance.